pkg/sloop/webserver/debug.go

```diff
@@ -162,6 +164,107 @@ func listKeysHandler(tables typed.Tables) http.HandlerFunc {
 	}
 }
 
+type SloopKeyInfo struct {
+	TotalKeys     int64
+	EstimatedSize int64
+}
+
+type SloopKey struct {
+	TableName   string
+	PartitionID string
+}
+
+type histogram struct {
+	HistogramMap map[SloopKey]*SloopKeyInfo
+	TotalKeys    int
+	DeletedKeys  int
+}
+
+// returns IsDeletedExpired, TableName, PartitionId, EstimatedSize, error.
+func ParseSloopKey(item badgerwrap.Item) (bool, string, string, int64, error) {
+	key := item.Key()
+	parts := strings.Split(string(key), "/")
```

Comment:
this logic (key split, verify 6 parts, 1st part is empty) exists in 4 other places - consider creating a shared method instead of duplicating the logic again